app/templates/road-sign-concepts/new.hbs

```diff
@@ -7,51 +7,75 @@
     </LinkTo>
   </BreadcrumbsItem>
 
-  <NavigationBar />
+  <NavigationBar/>
 
   <form
-    class="au-c-form au-o-layout au-u-padding-top-large"
+          class="au-c-form au-o-layout au-u-padding-top-large"
     {{on "submit" this.createRoadSignConcept}}
   >
     <p>
-      <AuLabel for="image-url">{{t "newRoadSign.imageUrl"}}</AuLabel>
-      <AuInput type="url" id="image-url" required="required" {{on "input" (fn this.setRoadSignConceptValue "image")}} />
+      <AuLabel for="image-url">{{t "newRoadSign.imageUrl"}}&nbsp;<AuPill>{{t "newRoadSign.required"}}</AuPill>
```

Comment:
Maybe you can wrap everything inside a let block:

```hbs
{{#let (is-empty @model.newRoadSignConcept.image) as |isInvalid|}}
{{!-- use isInvalid --}}
{{/let}}
```

That would reduce the duplication and make it more clear what's happening?

Reply:
I think the AuLabel also has an `@error` argument, but maybe that's overkill?

Reply:
there is no @error on the aulabel level. I tried it and it doesn't do anything 
![image](https://user-images.githubusercontent.com/3816305/107482500-0e031100-6b80-11eb-8090-54cf8bc8618d.png)